### PR TITLE
Fix possible mistake in activity-counts-metrics-plugin.md

### PR DESCRIPTION
**Pull request by ep3p**
I believe there was a mistake copypasting from new_activity_metrics plugin

Please, could you check?

### CODE REVIEW

**Comment by Court72** [comment]
@shsagir

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: data-explorer/kusto/query/activity-counts-metrics-plugin.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ T | evaluate activity_counts_metrics(id, datetime_column, startofday(ago(30d)),
 | *TimelineColumn* | string | &check; | The name of the column that represents the timeline. |
 | *Start* | datetime | &check; | The analysis start period. |
 | *End* | datetime | &check; | The analysis end period. |
-| *LoockbackWindow* | decimal, datetime, or timespan | &check; | The analysis window period. The value may also be a string of `week`, `month`, or `year`, in which case all periods would be [startofweek](startofweekfunction.md), [startofmonth](startofmonthfunction.md), or [startofyear](startofyearfunction.md). |
+| *LookbackWindow* | decimal, datetime, or timespan | &check; | The analysis window period. The value may also be a string of `week`, `month`, or `year`, in which case all periods would be [startofweek](startofweekfunction.md), [startofmonth](startofmonthfunction.md), or [startofyear](startofyearfunction.md). |
```
Comment (yifatsor) [inline data-explorer/kusto/query/activity-counts-metrics-plugin.md:29]: should be called either `Step` or `Window` like in `new_activity_metrics` or `activity_metrics`. Lookback is not the right term to use.

Comment (ep3p) [reply data-explorer/kusto/query/activity-counts-metrics-plugin.md]: @yifatsor substituted ```LookbackWindow``` by ```Window```

**Comment by YaelSchuster** [comment]
Thanks for your suggestion.
I've created a work item for internal tracking and follow up.

#please-close

**Pull request closed without merging**